src/client/screens/HomeScreen.tsx

```diff
@@ -1,35 +1,118 @@
 /** @jsx jsx */
+import Menu, { MenuItem, SubMenu } from "rc-menu";
+import "rc-menu/assets/index.css";
 import React, { useEffect } from "react";
+import Avatar from "react-avatar";
 import { connect } from "react-redux";
 import { Link } from "react-router-dom";
-import { Box, Divider, Flex, Heading, jsx, Styled } from "theme-ui";
+import { Box, Button, Divider, Flex, Heading, jsx, Styled } from "theme-ui";
 
+import { IUser } from "../../shared/entities";
+import { resetState } from "../actions/root";
 import { projectsFetch } from "../actions/projects";
+import { userFetch } from "../actions/user";
+import { clearJWT, getJWT } from "../jwt";
 import { State } from "../reducers";
 import { ProjectsState } from "../reducers/projects";
+import { Resource } from "../resource";
 import store from "../store";
+import theme from "../theme";
```

Comment:
`Button` and `theme` aren't being used.

```suggestion
import { Box, Divider, Flex, Heading, jsx, Styled } from "theme-ui";

import { IUser } from "../../shared/entities";
import { resetState } from "../actions/root";
import { projectsFetch } from "../actions/projects";
import { userFetch } from "../actions/user";
import { clearJWT, getJWT } from "../jwt";
import { State } from "../reducers";
import { ProjectsState } from "../reducers/projects";
import { Resource } from "../resource";
import store from "../store";
```